<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SiteItem</class>
 <widget class="QWidget" name="SiteItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(38, 68, 83, 92);
color: rgba( 64, 230, 64, 255);</string>
  </property>
  <widget class="QWidget" name="site_item_wdg" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>831</width>
     <height>205</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit: {
background-color: rgb(32, 74, 135);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="name_txbx">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>IPNS key local name (only exists in the context of this computer)</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0.298969 rgba(0, 0, 0, 40), stop:1 rgba(0, 0, 0, 30));</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="remove_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>5</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>delete this Site and its IPNS key</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(102, 10, 10);
color: rgb(252, 233, 79);</string>
         </property>
         <property name="text">
          <string>â€“</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_7">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="src_path_btn">
            <property name="text">
             <string>source path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="toolTip">
             <string>the IPFS CID of the currently published version of the Site, changes with every update</string>
            </property>
            <property name="text">
             <string>IPFS CID</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="toolTip">
             <string>IPNS key of this site, never changes</string>
            </property>
            <property name="text">
             <string>IPNS</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="path_txbx">
            <property name="toolTip">
             <string>source file/folder for this Site</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(32, 74, 135);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ipfs_cid_txbx">
            <property name="toolTip">
             <string>the IPFS CID of the currently published version of the Site, changes with every update</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(32, 74, 135);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ipns_key_id_txbx">
            <property name="toolTip">
             <string>IPNS key of this site, never changes</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(32, 74, 135);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="update_txbx">
            <property name="toolTip">
             <string>update this Site from the source path, uploading the source to IPFS and assigning it to this IPNS key</string>
            </property>
            <property name="text">
             <string>Update from Source</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="open_cid_txbx">
            <property name="toolTip">
             <string>open this Site in your browser using the IPFS CID</string>
            </property>
            <property name="text">
             <string>Open in Browser</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="open_txbx">
            <property name="toolTip">
             <string>open this Site in your browser using the IPNS key</string>
            </property>
            <property name="text">
             <string>Open in Browser</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
